scripts/install.sh

```diff
@@ -3,8 +3,15 @@
 set -e
 set -o pipefail
 
+RUSTUP_COMMAND="curl https://sh.rustup.rs -sSf | sh -s -- -y"
+if [ ! -z $CI ]; then
+    default_compiler=`cat rust-toolchain`
+    echo "Setting the default compiler to $default_compiler"
+    RUSTUP_COMMAND="$RUSTUP_COMMAND --default-toolchain $default_compiler"
+fi
 echo "Installing Rust toolchain..."
-curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain $(cat rust-toolchain)
+eval $RUSTUP_COMMAND
```

Comment:
Not sure if this one should be quoted as well?

Reply:
It's a good question and I'm not sure as well - all the examples I found on the internet were unquoted - but haven't found anything definitive. This definitely works now and it contains spaces already, so maybe eval does not care ? I will change it to quoted one for consistency though as it seems to work as well.

Reply:
If it works and it's ok, we can keep it. Was just asking, not sure here ...